README.md

```diff
@@ -54,6 +56,88 @@ module.exports = {
 
 Please update `testMatch` to match your project folder structure
 
+#### Alongside other runners
```

Comment:
This is awesome! I think as a next step it would be awesome to make a PR to `facebook/jest` and copy this as the documentation for Jest Projects. Once that happens we can just link to those docs from here